File: SecondYear/semestrul4/MPP/examen_practic/Rezolvari/SUB3_2024/networking/src/main/java/org/networking/rpcprotocol/RequestType.java
package org.networking.rpcprotocol;

import java.io.Serializable;

public enum RequestType implements Serializable {
    LOGIN,
    LOGOUT,
    GET_ORGANIZATORI,
    ADD_GAME,
    GET_ALL_GAMES,
    FIND_PLAYER_BY_ALIAS,
    GET_ALL_CONFIGURATIONS
}

File: SecondYear/semestrul4/MPP/PROIECT/CSharpProjectNetworking/Networking/rpcprotocol/RequestType.cs


namespace Networking.rpcprotocol;

[Serializable]
public enum RequestType
{
    LOGIN, 
    LOGOUT,
    GET_PROBE, 
    GET_ORGANIZATORI, 
    SEARCH_PARTICIPANTS, 
    GET_PROBE_DTO,
    GET_PROBA_BY_NAME,
    GET_PROBA_BY_NAME_AND_RANGE, 
    GET_VARSTA_BY_RANGE,
    FIND_PARTICIPANT_BY_CNP, 
    REFRESH_PROBE_STATISTICS,
    REGISTER
}
